install.sh

```diff
@@ -295,42 +285,45 @@ for i in $(seq 0 $((${#urls[@]} - 1))); do
   tarfile=$(basename "${url}")
   name="${tarfile%%-*}"   # extract string before first '-'
   rest="${tarfile#*-}"    # extract string after first '-'
-  version="${rest%%-chromeos*}"
-                          # extract string between first '-' and "-chromeos"
+  version="${rest%%-chromeos*}" # extract string between first '-' and "-chromeos"
 
   download_check "${name}" "${url}" "${tarfile}" "${sha256}"
   extract_install "${name}" "${tarfile}"
   update_device_json "${name}" "${version}"
 done
 
-echo_info "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
-echo -e "${GRAY}"
+# Work around https://github.com/chromebrew/chromebrew/issues/3305.
+# shellcheck disable=SC2024
+sudo ldconfig &> /tmp/crew_ldconfig || true
```

Comment:
Why is `sudo` needed here?

Reply:
idk, I didn't write that line

Reply:
I presume it's just because ldconfig needs sudo to run, as that's the case on other distros